MdeModulePkg/Library/UefiBootManagerLib/BmBoot.c

```diff
@@ -2715,3 +2715,57 @@ EfiBootManagerGetNextLoadOptionDevicePath (
 {
   return BmGetNextLoadOptionDevicePath (FilePath, FullPath);
 }
+
+//
+// MU_CHANGE begin
+//
+
+/**
+   Constructor for UefiBootMangerLib
+

```

Comment:
nit: Should have parameter and return status. It's an internal function though, so I don't feel strongly it must be added.